README.md

```diff
@@ -31,7 +31,9 @@ Other clients:
 
 ## Data Processing
 
-- [pulsar-flink](https://github.com/apache/pulsar/tree/master/examples/flink/src/main/java/org/apache/flink/batch/connectors/pulsar/example): A collection of streaming and batch connectors for [Apache Flink](http://flink.apache.org) processing streams in Pulsar. [batch](https://github.com/apache/pulsar/tree/master/examples/flink/src/main/java/org/apache/flink/batch/connectors/pulsar/example) and
+- Pulsar Flink Integration:
+  - [Flink 1.9+ Integration](https://github.com/streamnative/pulsar-flink): Streaming source and sink connector, Streaming Table support and catalog/schema integration.
+  - [Flink 1.6 Integration](https://github.com/apache/pulsar/tree/master/examples/flink/src/main/java/org/apache/flink/batch/connectors/pulsar/example): A collection of streaming and batch connectors for [Apache Flink](http://flink.apache.org) processing streams in Pulsar. [batch](https://github.com/apache/pulsar/tree/master/examples/flink/src/main/java/org/apache/flink/batch/connectors/pulsar/example) and
   [streaming](https://github.com/apache/pulsar/tree/master/examples/flink/src/main/java/org/apache/flink/streaming/connectors/pulsar/example).
 - Pulsar Spark Integration:
   - [pulsar-spark](http://pulsar.apache.org/docs/en/adaptors-spark/): A [Spark Streaming](http://spark.apache.org) receiver to receive data from Pulsar.
```

Comment:
is it correct?

```suggestion
  - [pulsar-spark](http://pulsar.apache.org/docs/en/adaptors-spark/): A [Spark Streaming](http://spark.apache.org) connector to read data from and write data to Pulsar.
```

Reply:
that's correct. that one is a receiver which only receive data. it doesn't write data.